service/templates/projects.html

```diff
@@ -976,102 +976,103 @@ <h3>Periodic Builds</h3>
-              <i class="fa fa-plus"></i>
-            </button>
-            <label class="distro-error">[[invalidContainerSizeMessage]]</label>
-          </div>
-        </div>
-      </div>
-      <!-- Container credentials used for private image repos -->
-      <div class="variables">
-        <div class="form-group">
-          <div class="col-header col-lg-6 form-control-static">
-            <h3>Container Credentials</h3>
-            <div class="muted small">Set username/password pairs to be used for authentication when interacting with private image repositories.</div>
-          </div>
-        </div>
-        <div id="container-credentials-list-header" class="form-group">
-          <div class="col-lg-2"> <label class="control-label"> Name </label> </div>
-          <div class="col-lg-2"> <label class="control-label"> Username </label> </div>
-          <div class="col-lg-2"> <label class="control-label"> Password </label> </div>
-          <div class="col-lg-2"></div>
-        </div>
-        <div id="container-credentials-list" class="form-group" ng-repeat="(name, obj) in settingsFormData.container_credentials track by $index">
-          <div class="col-lg-2">
-            <input class="form-control" ng-model="name" type="text" placeholder="name">
-          </div>
-          <div class="col-lg-2">
-            <input class="form-control" ng-model="obj.username" type="text" placeholder="username">
-          </div>
-          <div class="col-lg-2">
-            <input class="form-control" ng-model="obj.password" type="text" placeholder="password">
-          </div>
-          <div class="col-lg-2">
-            <button class="btn btn-default btn-danger" type="button" ng-click="removeContainerCredential(name)">
-              <i class="fa fa-trash"></i>
-            </button>
-          </div>
-        </div>
-        <div class="form-group">
-          <div class="col-lg-2">
-            <input ng-model="container_credential.name" class="form-control" type="text" placeholder="name">
-          </div>
-          <div class="col-lg-2">
-            <input ng-model="container_credential.username" class="form-control" type="text" placeholder="username">
-          </div>
-          <div class="col-lg-2">
-            <input ng-model="container_credential.password" class="form-control" type="text" placeholder="password">
-          </div>
-          <div class="col-lg-2">
-            <button class="plus-button btn btn-primary " ng-disabled="!validContainerCredential(container_credential)" type="button" ng-click="addContainerCredential()">
-              <i class="fa fa-plus"></i>
-            </button>
-            <label class="distro-error">[[invalidContainerCredentialMessage]]</label>
-          </div>
-        </div>
-      </div>
+<!--      TODO: EVG-17134 Uncomment once container tasks are available-->
```

Comment:
An alternative way to do this would be to wrap the whole thing in a div that has hide=true, which may be cleaner